lib/responders/flash_responder.rb

```diff
@@ -153,13 +153,22 @@ def mount_i18n_options(status) #:nodoc:
         :downcase_resource_name => resource_name.downcase
       }
 
-      if controller.respond_to?(:interpolation_options, true)
-        options.merge!(controller.send(:interpolation_options))
+      controller_options = controller_interpolation_options
+      if controller_options
+        options.merge!(controller_options)
```

Comment:
Why do you merge it here again, if you are already merging at the `controller_interpolation_options` ?